src/pages/workspaces/workspace/Data.js

```diff
@@ -551,7 +552,10 @@ const DeleteObjectModal = ajaxCaller(class DeleteObjectModal extends Component {
   }
```

Comment:
I saw on the ticket that one of the AC is for downloading files from data, and I don't see that here. Is that a follow-on? Or captured elsewhere?

Reply:
That will require some Martha work, so it will get addressed as part of SATURN-861